### ISSUE TITLE
node.js exception when giving the hyper.log function non-character data.

### ISSUE THREAD

**Issue by Sionarch** [evothings/evothings-studio#62]
When on the mobile device giving the hyper.log function non-character data, e.g. some UTF-8 encoded binary data, a node.js exception occurs in the Workbench. The following error message is shown in the Workbench window.

```
Uncaught node.js Error 

Error: Failed to execute 'postMessage' on 'Window': An object could not be cloned.
    at Error (native)
    at file:///Applications/EvothingsStudio_Mac_64_1.1.3/hyper/ui/hyper-ui.js:476:22
    at displayLogMessage (/Applications/EvothingsStudio_Mac_64_1.1.3/hyper/server/hyper-server.js:717:3)
    at Socket.<anonymous> (/Applications/EvothingsStudio_Mac_64_1.1.3/hyper/server/hyper-server.js:780:4)
    at Socket.EventEmitter.emit (events.js:104:17)
    at Socket.onevent (/Applications/EvothingsStudio_Mac_64_1.1.3/node_modules/socket.io/lib/socket.js:327:8)
    at Socket.onpacket (/Applications/EvothingsStudio_Mac_64_1.1.3/node_modules/socket.io/lib/socket.js:291:12)
    at Client.ondecoded (/Applications/EvothingsStudio_Mac_64_1.1.3/node_modules/socket.io/lib/client.js:185:14)
    at Decoder.Emitter.emit (/Applications/EvothingsStudio_Mac_64_1.1.3/node_modules/component-emitter/index.js:134:20)
    at Decoder.add (/Applications/EvothingsStudio_Mac_64_1.1.3/node_modules/socket.io-parser/index.js:257:14)
```

**Comment by fredrikeldh** [comment]
We need to know what kind of object this is, that can not be cloned. You may test using hyper.log() and NODE EVAL in the Tools window.

**Comment by divineprog** [comment]
Passing in function objects causes this crash when doing EVAL on a device. NODE EVAL works.

**Comment by fredrikeldh** [comment]
The definition of hyper.log is found in HyperReload, [here](https://github.com/evothings/HyperReload/blob/4114fb53e48615d2d014713a3e38c518890ab08c/hyper/server/hyper-reloader.js#L46). It should be easily fixable by wrapping ```message``` in a ```String()```.

**Comment by fredrikeldh** [comment]
Fixed in commit 198749f3e38f36e8d117ef67ecadc31f302177dd.
